coopiteasy_custom/models/analytic_account.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from datetime import date, timedelta
+
+from odoo import api, fields, models
+
+
+class AccountAnalyticAccount(models.Model):
+    _inherit = "account.analytic.account"
+    _order = "line_count desc"
+
+    line_count = fields.Integer(
+        string="Line Count", compute="_compute_line_count", store=True
+    )
+
+    @api.depends("line_ids")
+    @api.multi
+    def _compute_line_count(self):
+        # may lead to performance issues. See with time.
+        for account in self:
+            one_month_ago = date.today() - timedelta(days=30)
+            recent_lines = account.line_ids.filtered(
+                lambda l: l.date >= one_month_ago
+            )
+            account.line_count = len(recent_lines)
```

Comment:
This variable is used nowhere. What is this for?

Reply:
is is used in the field `_order` at line 12. It is used to sort the account by "most used recently" : very very handy when logging time. So handy it could be proposed to the OCA.

Reply:
This will not be included in Komunigi. Komunigi only treats modules of vertical-cooperative and obeesdoo. If you have spare time on your migration time, you can submit to the OCA, otherwise, no worries.